ceno_zkvm/src/instructions/riscv/slt.rs

```diff
@@ -168,7 +226,30 @@ mod test {
         let a: i32 = rng.gen();
         let b: i32 = rng.gen();
         println!("random: {} <? {}", a, b); // For debugging, do not delete.
-        verify("random 1", a, b, (a < b) as u32);
-        verify("random 2", b, a, (a >= b) as u32);
+        verify::<SltOp>("random 1", a as Word, b as Word, (a < b) as u32);
+        verify::<SltOp>("random 2", b as Word, a as Word, (a >= b) as u32);
+    }
+
+    #[test]
+    fn test_sltu_simple() {
+        verify::<SltuOp>("lt = true, 0 < 1", 0, 1, 1);
+        verify::<SltuOp>("lt = true, 1 < 2", 1, 2, 1);
+        verify::<SltuOp>("lt = true, 0 < u32::MAX", 0, u32::MAX, 1);
+        verify::<SltuOp>("lt = true, u32::MAX - 1", u32::MAX - 1, u32::MAX, 1);
+        verify::<SltuOp>("lt = false, u32::MAX", u32::MAX, u32::MAX, 0);
+        verify::<SltuOp>("lt = false, u32::MAX - 1", u32::MAX, u32::MAX - 1, 0);
+        verify::<SltuOp>("lt = false, u32::MAX > 0", u32::MAX, 0, 0);
+        verify::<SltuOp>("lt = false, 2 > 1", 2, 1, 0);
+    }
+
+    #[test]
+    fn test_sltu_random() {
+        // TODO(Matthias): use property pased testing.
```

Comment:
Copy and pasting TODOs should perhaps be a warning sign that we are doing too much copy and pasting?

Reply:
A proposal of a better testing framework should track in separate issue and clean up the comment here, because this is not an issue specific to slt opcode testing

Reply:
I've just created an issue https://github.com/scroll-tech/ceno/issues/687

> Copy and pasting TODOs should perhaps be a warning sign that we are doing too much copy and pasting?

In context of this PR it is moving the code not duplicating, but agreed.